src/Symfony/Component/Console/Helper/TableStyle.php

```diff
@@ -108,18 +117,71 @@ public function getVerticalBorderChar()
         return $this->verticalBorderChar;
     }
 
+    /**
+     * Sets crossing characters.
+     *
+     * Example:
+     * <code>
+     * 1---------------2-----------------------2------------------3
+     * | ISBN          | Title                 | Author           |
+     * 8---------------0-----------------------0------------------4
+     * | 99921-58-10-7 | Divine Comedy         | Dante Alighieri  |
+     * | 9971-5-0210-0 | A Tale of Two Cities  | Charles Dickens  |
+     * | 960-425-059-0 | The Lord of the Rings | J. R. R. Tolkien |
+     * 7---------------6-----------------------6------------------5
+     * </code>
+     *
+     * @param string $cross       Crossing char (see #0 of example)
+     * @param string $topLeft     Top left char (see #1 of example)
+     * @param string $topMid      Top mid char (see #2 of example)
+     * @param string $topRight    Top right char (see #3 of example)
+     * @param string $midRight    Mid right char (see #4 of example)
+     * @param string $bottomRight Bottom right char (see #5 of example)
+     * @param string $bottomMid   Bottom mid char (see #6 of example)
+     * @param string $bottomLeft  Bottom left char (see #7 of example)
+     * @param string $midLeft     Mid left char (see #8 of example)
+     *
+     * @return $this
```

Comment:
Can  be removed as the method signature already document this explicitly.

Reply:
`<code>` tag is sometimes used yes (on master, 17 occurrences). This code-description can be useful IMO.

Reply:
Done